api/src/utils/mapperUtils.ts

```diff
@@ -1,9 +1,9 @@
-import { Project } from 'src/database/entities/Project';
-import { Reply } from 'src/database/entities/Reply';
-import { ReplySnippet } from 'src/database/entities/ReplySnippet';
-import { Thread } from 'src/database/entities/Thread';
-import { ThreadSnippet } from 'src/database/entities/ThreadSnippet';
-import { User } from 'src/database/entities/User';
+import { Project } from '../database/entities/Project';
+import { Reply } from '../database/entities/Reply';
+import { ReplySnippet } from '../database/entities/ReplySnippet';
+import { Thread } from '../database/entities/Thread';
+import { ThreadSnippet } from '../database/entities/ThreadSnippet';
+import { User } from '../database/entities/User';
```

Comment:
I actually like absolute import paths over relative import paths. It's clearer and also helps when directories are updated. Here is [doc](https://dev.to/tasawarhussain/react-clean-code-with-absolute-imports-1028#) listing the positives of absolute path.

Reply:
I know it's kinda controversial as relative paths has it's own positives but I think absolute paths helps to visually identify from which folder/path it's coming from.

Reply:
According to google typescript guide, we should use relative https://google.github.io/styleguide/tsguide.html#imports

I don't have any preference over another. But we have this guide in our contributing guideline which helps us decide certain syntax choices.

Reply:
How would you feel if we probably add an exception to this rule? or I guess I can live with it.

Reply:
Will need to change the config, not worth it imo.

Reply:
cool let's leave it then.